components/engine/libcontainerd/client_linux.go

```diff
@@ -330,6 +332,14 @@ func (clnt *client) restore(cont *containerd.Container, lastEvent *containerd.Ev
 		return fmt.Errorf("container %s is already active", containerID)
 	}
 
+	if errUpgrade := v17_06_1.Upgrade(
```

Comment:
thanks, this is not supposed to be here... only the other Upgrade() call. Will remove.